### ISSUE TITLE
glx2scripteditor.rev is in two places in repo

### ISSUE THREAD

**Issue by macMikey** [mwieder/glx2ScriptEditor#17]
you have it in both the `/` and in the `/place these in your liveCode User Plugins folder`

**Comment by mwieder** [comment]
Yes - that's necessary because you can't download the raw binary from inside the folder.
Took quite a bit of thrashing about here to get to the bottom of that.

**Comment by mwieder** [comment]
I could get rid of the folder and go back to way things were, but the folder seems like a good idea.

**Comment by macMikey** [comment]
I think what you should do is get rid of the copy that is not in the folder.

**Comment by mwieder** [comment]
I'll backburner this for now - it would mean changing the update check routine.
